CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ The most noteworthy change of this release is the update of the container's base
 ### Fixes
 
 - DMS config files that are parsed line by line are now more robust to parse by detecting and fixing line-endings ([#3819](https://github.com/docker-mailserver/docker-mailserver/pull/3819))
+- variables related to Rspamd are declared as `readonly`, which would cause warnings in the log when being re-declared; we now guard agains this issue ([#3837](https://github.com/docker-mailserver/docker-mailserver/pull/3837))
```

Comment:
```suggestion
- Variables related to Rspamd are declared as `readonly`, which would cause warnings in the log when being re-declared; we now guard against this issue ([#3837](https://github.com/docker-mailserver/docker-mailserver/pull/3837))
```